lib/main.dart

```diff
@@ -18,6 +19,13 @@ void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   final storage = loadFromLocalStorage();
   final firebase = Firebase.initializeApp(options: DefaultFirebaseOptions.currentPlatform);
+  HardwareKeyboard.instance.addHandler((event) {
+    if (event is KeyDownEvent && event.logicalKey == LogicalKeyboardKey.escape) {
+      navigator.pop();
+      return true;
+    }
+    return false;
+  });
```

Comment:
My Android phone has a "back button" that helps with navigating through apps.

With this keyboard event handler, hitting "escape" will do the same thing.